weasyl/message.py

```diff
@@ -3,6 +3,7 @@
 from weasyl import character
 from weasyl import define as d
 from weasyl import media
+from weasyl import searchtag as st
```

Comment:
Don't rename this, imo, since it's non-obvious what 'st' stands for and it's only used in a few places.